src/client/routes/Market/index.tsx

```diff
@@ -140,7 +153,7 @@ export const Market = () => {
               key={key}
               items={val.map(({ id, borrower, spigot, escrow, principal, deposit, start, end }) => ({
                 icon: '',
-                name: borrower,
+                name: (getENS(borrower, ensMap) ? getENS(borrower, ensMap) : borrower)!,
```

Comment:
`ensMap[borrower] ?? borrower` or make `getENS` a hook `useENS(address)`that does the same. Hook is preferable since we'll use this a lot.